### ISSUE TITLE
WS connection error 

### ISSUE THREAD

**Issue by TheCoinCom** [orionprotocol/trading-sdk#24]
Suddenly, the sdk responds with "WS connection error - check your path" when connecting to basically anything. Price listener or balance checkers. No changes were made to the code. The error suddenly appeared.

**Comment by virus-zip** [comment]
Yes, this still sometimes comes up. Could it be a server side timeout where the WS cancels the connection after a certain period of time?
